packages/web/components/Player/PlayerRank.tsx

```diff
@@ -42,7 +41,7 @@ export const PlayerRank: React.FC<PlayerRankProps> = ({
       {showSeasonalXP && (
         <Text fontSize="sm" color="blueLight">
           {/* hardcoded Season #, will have to be changed every season */}
-          S8: {Math.floor(player.seasonXP).toLocaleString()}
+          SⅧ: {Math.floor(player.seasonXP).toLocaleString()}
```

Comment:
this can be achieved by using 
```
import { convertToRoman } from 'utils/formatHelpers';
import { getSeasonNum, Maybe } from '@metafam/utils';
...
{convertToRoman(getSeasonNum())}: {Math.floor(player.seasonXP).toLocaleString()}
````